.gitignore

```diff
@@ -355,3 +355,5 @@ MigrationBackup/
 /tests/E2E Tests/PlaywrightTraces
 /tests/IntegrationTests/PlaywrightTraces
 /.SharedData
+/out/
```

Comment:
these folders are created by newest versions of msbuild (when ran on the command line)